packages/poaps/src/queries/PaginatedPoaps.ts

```diff
@@ -0,0 +1,49 @@
+export const PAGINATED_POAPS_QUERY = `
+  query PaginatedPoaps(
+    $limit: Int!
+    $offset: Int!
+    $orderBy: [poaps_order_by!]
+    $where: poaps_bool_exp
+  ) {
+    poaps(limit: $limit, offset: $offset, order_by: $orderBy, where: $where) {
+      chain
+      collector_address
+      drop_id
+      id
+      minted_on
```

Comment:
Is this called `created_date` or `minted_on`? I am missing something I think.

Reply:
It's `minted_on`

Reply:
:+1: